cross/librsync/Makefile

```diff
@@ -1,17 +1,67 @@
 PKG_NAME = librsync
-PKG_VERS = 0.9.7
+PKG_VERS = 2.2.1
 PKG_EXT = tar.gz
 PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
-PKG_DIST_NAME = $(PKG_DIR).$(PKG_EXT)
-PKG_DIST_SITE = http://downloads.sourceforge.net/project/librsync/$(PKG_NAME)/$(PKG_VERS)
+PKG_DIST_FILE = $(PKG_DIR).$(PKG_EXT)
+PKG_DIST_NAME = v$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://github.com/librsync/$(PKG_NAME)/archive
 
```

Comment:
For the sake of order please define PKG_DIR as last variable after PKG_DIST_SIZE.
1. PKG_DIST_NAME = the file to download
2. PKG_DIST_SITE = the URL (without PKG_DIST_NAME) to download the file $(PKG_DIST_NAME)
3. define PKG_DIST_FILE only, when PKG_DIST_NAME does not contain the PKG_NAME (i.e. version only) to avoid conflicts of downloaded files in the distrib folder
4. PKG_DIR = the name of the folder that results when the downloaded file is extracted

Reply:
:+1: fixed